openstack/resource_openstack_blockstorage_quotaset_v3.go

```diff
@@ -101,30 +101,43 @@ func resourceBlockStorageQuotasetV3Create(ctx context.Context, d *schema.Resourc
 		return diag.Errorf("Error creating OpenStack block storage client: %s", err)
 	}
 
+	updateOpts := quotasets.UpdateOpts{}
 	projectID := d.Get("project_id").(string)
-	volumes := d.Get("volumes").(int)
-	snapshots := d.Get("snapshots").(int)
-	gigabytes := d.Get("gigabytes").(int)
-	perVolumeGigabytes := d.Get("per_volume_gigabytes").(int)
-	backups := d.Get("backups").(int)
-	backupGigabytes := d.Get("backup_gigabytes").(int)
-	groups := d.Get("groups").(int)
+	if v, ok := getOkExists(d, "fixed_ips"); ok {
```

Comment:
You have a typo here
```suggestion
	if v, ok := getOkExists(d, "volumes"); ok {
```

Reply:
Right... Thanks for spotting and fixing it